packages/filebrowser-extension/src/index.ts

```diff
@@ -1103,6 +1105,17 @@ function addCommands(
     label: trans.__('Duplicate')
   });
 
+  commands.addCommand(CommandIDs.selectAll, {
+    execute: () => {
+      const widget = tracker.currentWidget;
+
+      if (widget) {
+        return widget.selectAll();
+      }
+    },
+    label: trans.__('Select All')
```

Comment:
Picking up `describedBy` noise from #17649.

```suggestion
    label: trans.__('Select All'),
    describedBy: {
        args: {
          type: 'object',
          properties: {}
        }
      },
```